wpimath/src/test/native/cpp/filter/DebouncerTest.cpp

```diff
@@ -2,47 +2,51 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
-#include "frc/Debouncer.h"  // NOLINT(build/include_order)
+#include <wpi/timestamp.h>
 
-#include "frc/simulation/SimHooks.h"
+#include "frc/filter/Debouncer.h"
 #include "gtest/gtest.h"
+#include "units/time.h"
 
-using namespace frc;
+static units::second_t now = 0_s;
 
 TEST(DebouncerTest, DebounceRising) {
-  Debouncer debouncer{20_ms};
+  WPI_SetNowImpl([] { return units::microsecond_t{now}.to<uint64_t>(); });
```

Comment:
This will break other tests unless it is reset back to the default implementation.  Use a fixture to do this.

Reply:
You can use wpi::SetNowImpl(nullptr) to do this.